staff/marti-malek/gamma-template/about.html

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>About</title>
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.6.3/css/all.css" integrity="sha384-UHRtZLI+pbxtHCWp1t77Bi1L4ZtiqrqD80Kn4Z8NTSRyMA2Fd33n5dQ8lWUE00s/" crossorigin="anonymous">
+    <link rel="stylesheet" href="style/style.css">
+</head>
+<body>
+    <header class="about-h">
+        <div>
+            <h2>Gamma</h2>
+            <nav>
+                <a href="./home.html">Home</a>
+                <a href="./about.html">About</a>
+                <a href="./contact.html">Contact</a>
+            </nav>
+        </div>
+        <button><i class="fas fa-bars"></i></button>
+    </header>
+    <main>
+        <div class="about-t"><h1>About Gamma</h1></div>
+        <div class="about-grid">
+            <section class="about-first">
+                <div>
```

Comment:
Do you need that div? Maybe with the section tag is enough?

Reply:
I'm referring to it in style.sass